src/com/chessclock/android/ChessClock.java

```diff
@@ -191,12 +198,25 @@ private int initTime(int player) {
             : initTime1;
     }
 
+    /** Return time in milliseconds based on the specified unit. */
+    private int timeToMilliseconds(int time, String timeUnit) {
```

Comment:
How about `timeToMillis` to make it a bit less verbose (or maybe even just `toMillis`)? Not very important to me though, so let's keep it like that if you'd like. :)

Reply:
`toMillis` sounds good :+1: